htdocs/product/stock/tpl/stocktransfer.tpl.php

```diff
@@ -63,7 +63,7 @@
 		{
 		    print '<td width="15%" class="fieldrequired">'.$langs->trans("WarehouseSource").'</td>';
 		    print '<td width="15%">';
-		    print $formproduct->selectWarehouses((GETPOST("dwid")?GETPOST("dwid",'int'):(GETPOST('id_entrepot')?GETPOST('id_entrepot','int'):'ifone')),'id_entrepot','',1);
+		    print $formproduct->selectWarehouses((GETPOST("dwid")?GETPOST("dwid",'int'):(GETPOST('id_entrepot')?GETPOST('id_entrepot','int'):'ifone')),'id_entrepot_source','',1);
```

Comment:
id_entrepot moved into id_entrepot_source but we still have GETPOST('id_entrepot') ?